### PR TITLE
one load_data, add areacella weighted averaging

### PR DESCRIPTION
**Pull request by chiaral**

### CODE REVIEW

File: notebooks/mean_var_calcs.ipynb [hunk 1]
```diff
@@ -32,6 +32,7 @@
    "source": [
```
Comment (chiaral) [inline notebooks/mean_var_calcs.ipynb]: <p>I actually have to delete that extra experiment_ids1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/apendergrass/cmip6hack-precip/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (chiaral) [inline notebooks/mean_var_calcs.ipynb]: <p>the unique load function</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/apendergrass/cmip6hack-precip/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (chiaral) [inline notebooks/mean_var_calcs.ipynb]: <p>this is where I try to load areacella, and if it doesn't work, it gets a none flag.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/apendergrass/cmip6hack-precip/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (chiaral) [inline notebooks/mean_var_calcs.ipynb]: <p>my results don't get along with your code. not sure why</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/apendergrass/cmip6hack-precip/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (bolliger32) [reply notebooks/mean_var_calcs.ipynb]: <p>can we put this in <code>util.py</code>?</p>

**Pull request merged**